lib/tools/system-calls.js

```diff
@@ -411,6 +412,20 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   return proc;
 };
 
+systemCallMethods.getAdbVersion = async function () {
```

Comment:
rather than implementing our own caching can't we use lodash to just wrap the method? then we don't need an instance variable, too